conf/diaspora_sidekiq.service

```diff
@@ -3,29 +3,32 @@ Description=Diaspora social network (sidekiq - instance __APP__)
 PartOf=__APP__.target
 
 [Service]
+Type=simple
 User=__APP__
-Environment=RAILS_ENV=production
-WorkingDirectory=__FINALPATH__/diaspora
-ExecStart=/bin/bash -lc "bin/bundle exec sidekiq"
+WorkingDirectory=__FINALPATH__/live
+Environment="__LD_PRELOAD__"
+Environment="RAILS_ENV=production"
+Environment="__YNH_RUBY_LOAD_PATH__"
+ExecStart=__FINAL_PATH__/live/bin/bundle exec sidekiq
 Restart=always
 
 # Sandboxing options to harden security
 # Depending on specificities of your service/app, you may need to tweak these 
 # .. but this should be a good baseline
 # Details for these options: https://www.freedesktop.org/software/systemd/man/systemd.exec.html
-NoNewPrivileges=yes
+NoNewPrivileges=no
 PrivateTmp=yes
-PrivateDevices=yes
-RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
-RestrictNamespaces=yes
-RestrictRealtime=yes
+#PrivateDevices=yes
+#RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6 AF_NETLINK
+#RestrictNamespaces=yes
+#RestrictRealtime=yes
 DevicePolicy=closed
 ProtectSystem=full
 ProtectControlGroups=yes
-ProtectKernelModules=yes
-ProtectKernelTunables=yes
-LockPersonality=yes
-SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap
+#ProtectKernelModules=yes
+#ProtectKernelTunables=yes
+#LockPersonality=yes
+#SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap
```

Comment:
Can you elaborate why the changes in this file? You had trouble starting the server with the changes I did in 3e3cc64?